en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+`errorsDetailsFilter` parameter to further filter the results.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">errorsDetailsFilter</td>
+<td valign="top"><a href="#errorsdetailsfilter">ErrorsDetailsFilter</a></td>
+<td>
+
+Error details filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getProxyTargetErrorsOverTime</strong></td>
+<td valign="top">[<a href="#errorsbystatuscodecategory">ErrorsByStatusCodeCategory</a>]</td>
+<td>
+
+Returns a list of hit counts for proxy and target errors by each time interval (calculates according to the
+granularity value) within the specified time range for the given combination of tenant ID, environment ID, and
+organization ID. Each proxy and target error category result set contains breakdown of errors as 4xx, 5xx and total.
+Optionally, you can configure the `errorsByStatusCodeFilter` parameter to further filter the results.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">errorsByStatusCodeFilter</td>
+<td valign="top"><a href="#errorsbystatuscodefilter">ErrorsByStatusCodeFilter</a></td>
+<td>
+
+Errors by status code filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getErrorsByStatusCode</strong></td>
+<td valign="top"><a href="#errorsbystatuscode">ErrorsByStatusCode</a></td>
+<td>
+
+Returns hit count for each error response code (401, 404, 500, etc.) for each API within the specified time range
+for the given combination of tenant ID, environment ID, and organization ID.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">errorCountByStatusCodeFilter</td>
+<td valign="top"><a href="#errorcountbystatuscodefilter">ErrorCountByStatusCodeFilter</a></td>
+<td>
+
+Error count by status code filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getTotalErrorsByAPI</strong></td>
+<td valign="top"><a href="#totalerror">TotalError</a></td>
+<td>
+
+Returns the total proxy error hit count for an API across all the data available time range for the given
```

Comment:
The phrase "across all the data available time range" is unclear. Did you mean "across all the dates available within the selected time range"?

Reply:
simplified